spec/lib/saml_idp/request_spec.rb

```diff
@@ -264,6 +264,17 @@ module SamlIdp
           expect(subject.requested_vtr_authn_context).to eq(vtr)
         end
       end
+
+      context 'context that contains a VTR substring but is not a VTR' do
```

Comment:
would it be worthwhile to test that `requested_vtr_authn_context` doesn't return  `urn:gov:gsa:ac:classes:sp:PasswordProtectedTransport:duo` explicitly, since it is a valid authn context?

Reply:
Yeah, it probably doesn't hurt to throw a test in there for it. The VTR that I went with I thought looked more like a VTR to make the issue more obvious.